lib/ios/RNNUIBarButtonItem.m

```diff
@@ -125,6 +125,7 @@ - (void)applyOptions:(RNNButtonOptions *)buttonOptions {
     self.accessibilityLabel = [buttonOptions.accessibilityLabel withDefault:nil];
     self.enabled = [buttonOptions.enabled withDefault:YES];
     self.accessibilityIdentifier = [buttonOptions.testID withDefault:nil];
+    self.tintColor = [buttonOptions.color withDefault:nil];
```

Comment:
Seems like this might want to be `getWithDefaultValue:nil` instead of `withDefault:nil` -- ran into a compilation issue without it.

Reply:
I renamed `getWithDefaultValue` to `withDefault`  [here](4a5b8a4ac44c4a3c6a0c81fa13e8aaf284c4c983). The next RNN version will include this fix.

Reply:
Oh totally missed that, thank you!